figures/Comparison-ERP-TRF.py

```diff
@@ -0,0 +1,182 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: tutorialAlice
+#     language: python
+#     name: tutorialalice
+# ---
+
+import eelbrain
+import os
+from pathlib import Path
+import copy
+import numpy as np
+from matplotlib import pyplot
+import os
+
+# +
+# Data locations
+DATA_ROOT = Path("~").expanduser() / 'Data' / 'Alice'
+PREDICTOR_DIR = DATA_ROOT / 'predictors'
+STIMULI_DIR = DATA_ROOT / 'stimuli'
+TRF_DIR = DATA_ROOT / 'TRFs'
+EPOCH_DIR = DATA_ROOT / 'Epochs'
+
+tstart = -0.1
+tstop = 1
+# -
+
+# Where to save the figure
+DST = DATA_ROOT / 'figures'
+DST.mkdir(exist_ok=True)
+
+# get all subjects
+subjects = [subject for subject in os.listdir(TRF_DIR) if subject.startswith('S') ]
+assert os.path.exists(EPOCH_DIR), "Epoch directory is not found. Please, run script analysis/make-epochs.py to create the different epochs per subject."
+
+# (1) GET THE ERP RESPONSE TO A WORD ONSET
+cases = []
+for subject in subjects:
+    epochs = eelbrain.load.unpickle(EPOCH_DIR / subject / f'{subject}_epoched_word.pickle')
+    
+    # get erp and normalize amplitude
+    epochs['eeg'] -= epochs['eeg'].mean('time')
+    epochs['eeg'] /= epochs['eeg'].std('time')
```

Comment:
This is normalizing every single trial individually, is this really intended? Wouldn't it be more conventional to normalize the averaged ERP? It would also be more comparable to the TRF.